src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -477,6 +477,13 @@ public void throttle(double throttleValue) {
     throttleMultiplier = throttleValue;
   }
 
+  public void zeroSwerveAtCurrentPos() {
+    Cal.SwerveSubsystem.SWERVE_FRONT_LEFT_ANGULAR_OFFSET_RAD = frontLeft.getEncoderAbsPositionRad();
+    Cal.SwerveSubsystem.SWERVE_FRONT_RIGHT_ANGULAR_OFFSET_RAD = frontRight.getEncoderAbsPositionRad();
+    Cal.SwerveSubsystem.SWERVE_BACK_LEFT_ANGULAR_OFFSET_RAD = rearLeft.getEncoderAbsPositionRad();
+    Cal.SwerveSubsystem.SWERVE_BACK_RIGHT_ANGULAR_OFFSET_RAD = rearRight.getEncoderAbsPositionRad();
+  }
```

Comment:
We won't be able to zero all four at once, should be four different functions / buttons